site/src/App/Code/Code.treat.ts

```diff
@@ -0,0 +1,53 @@
+import { style } from 'sku/treat';
+
+export const code = style({
+  overflowX: 'auto',
+  borderBottomLeftRadius: '0 !important',
```

Comment:
Braid isn't currently good at this 😞

Reply:
Yeah, when i have faced this though rather than use `important` i have opted out of the box prop usage, eg. only apply border radius in the local treat file to prevent need for overriding. But not too fussed about the docs site.

Reply:
Might be a good idea to consider atomic border width classes.

Reply:
Or maybe a way of opting out of border radius at the `Box` level, since you're not really meant to mix and match different radius values.